website/pages/docs/download-tools.mdx

```diff
@@ -0,0 +1,64 @@
+- [files-to-consul-kv](https://github.com/bitsofinfo/files-to-consul-kv) - Ultra simple, configuration free CLI tool for syncing a directory structure of key-value files to Consul KV using the transactions API. Docker image available. Integrates easily into any CICD workflow.
+- [file2consul](https://github.com/joeatbayes/file2consul) - Update Consul values from git or files. Config loader with support for multiple environments. Provides variable expansion, interpolation, inheritance with overrides and ability to update multiple consul servers. Reduces cost of maintaining larger configuration sets between environments by reducing restatement and manual editing of similar or predictably changing config properties. MIT license, Written in GO.
+- [Flightpath](https://docs.flightpath.xyz/) - An xDS server that can configure Envoy to act as an Edge proxy for Consul Connect enabled services
+- [git2consul](https://github.com/ryanbreen/git2consul) - Mirror the contents of a Git repository into Consul KVs
+- [gobetween](https://github.com/yyyar/gobetween) - Modern & minimalistic load balancer and reverse-proxy for the ☁️ Cloud era.
+- [Gonsul](https://github.com/miniclip/gonsul) - A Git to Consul standalone tool made in Go. Updates Consul KV from a repo with multiple strategies.
+- [gradle-consul-plugin](https://github.com/amirkibbar/red-apple) - A Consul Gradle plugin
+- [hashi-ui](https://github.com/jippi/hashi-ui) - A modern user interface for the Consul and Nomad
+- [helios-consul](https://github.com/SVT/helios-consul) - Service registrar plugin for Helios
+- [Jenkins Consul Plugin](https://plugins.jenkins.io/consul) - Jenkins plugin for service discovery and K/V store
+- [marathon-consul](https://github.com/allegro/marathon-consul) - Service registry bridge for Marathon
+- [marathon-consul](https://github.com/CiscoCloud/marathon-consul) - Bridge from Marathon apps to the Consul K/V store
+- [mesos-consul](https://github.com/CiscoCloud/mesos-consul) - Service registry bridge for Mesos
+- [NodeFabric](http://opennodecloud.com/products/nodefabric.html) - Turnkey CentOS 7 Atomic Host image with integrated Consul, Registrator and HAProxy - enabling rapid MariaDB-Galera and Ceph deployments
+- [registrator](https://github.com/gliderlabs/registrator) - Service registry bridge for Docker
+- [Spring Cloud Consul](https://github.com/spring-cloud/spring-cloud-consul) - Service discovery, configuration and events for Spring Cloud
+- [IntelliJ IDEA Consul K/V Support](https://plugins.jetbrains.com/plugin/10511-consul-k-v-support) - editor for K/V store
+- [git2consul-go](https://github.com/KohlsTechnology/git2consul-go) - Data feeder for Consul K/V store
+
+Are you the author of a tool and you would like to be featured on this page? The Consul website is open source and is embedded inside the [Consul repository](https://github.com/hashicorp/consul) on GitHub. You can submit a Pull Request to add your tool to the list and we will gladly review it.
```

Comment:
This is duplicate content of `pages/downloads_tools` - if we want to move it here, we need to remove the other page, add a redirect, and change the link on the downloads page